services/datcore-adapter/src/simcore_service_datcore_adapter/api/routes/health.py

```diff
@@ -23,7 +23,7 @@
     status_code=status.HTTP_200_OK,
 )
 async def get_service_alive():
-    return f"{__name__}@{datetime.utcnow().isoformat()}"
+    return f"{__name__}@{datetime.now(timezone.utc).isoformat()}"
```

Comment:
no replace timezone here?

Reply:
Places, where the .repace(tzinfo=None) was added, were necessary because we later worked with the datetime that needed the format without tzinfo included inside. In the other cases (where .repace(tzinfo=None) was not included) it probably wasn't needed and it was ok to work with tzinfo included there (at least the unit tests were not complaining). But I can add it everywhere to be consistent if needed.

Reply:
maybe also check the feedback from @sanderegg about this, I see that he also has some doubts.
In theory I would prefer to be working with UTC times all across the place, if possible. I'm not sure if this is doable though.